### PR TITLE
test: add chocolatey workflow

### PR DESCRIPTION
**Pull request by hugorut**

### CODE REVIEW

File: .github/workflows/windows.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_wrapper: false
+      - name: Run Infracost
+        run: infracost breakdown --path=examples/terraform-directory/code --format=json --out-file=/tmp/infracost.json
+      - name: Post the comment
+        uses: infracost/actions/comment@v1
+        with:
+          path: /tmp/infracost.json
+          behavior: new
```
Comment (vdmgolub) [inline .github/workflows/windows.yml:32]: @hugorut We could use `infracost comment` instead of the comment action. What do you think?

Comment (hugorut) [reply .github/workflows/windows.yml:32]: I'm just testing

Comment (hugorut) [reply .github/workflows/windows.yml:32]: this isn't to add to the actions repo yet

Comment (vdmgolub) [reply .github/workflows/windows.yml:32]: Ah, I see 👍

**Pull request closed without merging**